tests/io/test_data_catalog.py

```diff
@@ -336,20 +337,14 @@ def test_catalogs_eq(self, data_catalog_from_config):
         data_catalog_copy = deepcopy(data_catalog_from_config)
         assert not data_catalog_copy == expected_catalog
 
-    def test_get_returns_none_dataset_raises(self):
+    def test_get_returns_none_dataset(self):
         catalog = DataCatalog(datasets={})
         catalog._datasets["bad_ds"] = None
```

Comment:
I don't think this line is necessary? It should work without affecting a bad_ds key, shouldn't it?